cql3/statements/list_service_level_statement.cc

```diff
@@ -58,7 +58,15 @@ list_service_level_statement::execute(query_processor& qp,
                 type);
     };
 
-    static thread_local const std::vector<lw_shared_ptr<column_specification>> metadata({make_column("service_level", utf8_type)});
+    static thread_local const std::vector<lw_shared_ptr<column_specification>> metadata({make_column("service_level", utf8_type),
+        make_column("read_timeout", duration_type),
+        make_column("write_timeout", duration_type),
+        make_column("range_read_timeout", duration_type),
+        make_column("counter_write_timeout", duration_type),
+        make_column("truncate_timeout", duration_type),
+        make_column("cas_timeout", duration_type),
+        make_column("other_timeout", duration_type),
```

Comment:
Having different timeouts for different operation types is dubious and I wish Cassandra hadn't done it.

Maybe we can live with just one timeout per session.

Reply:
I'd actually be very happy to rework this series to provide just a single timeout. This explosion is terrible and I only did it because we already have in scylla.yaml, but having just one per-session timeout would make this series so much simpler.

I'll go ahead and do it in v2 and see how it goes.